alembic/runtime/environment.py

```diff
@@ -367,7 +367,11 @@ def get_x_argument(
         The return value is a list, returned directly from the ``argparse``
         structure.  If ``as_dictionary=True`` is passed, the ``x`` arguments
         are parsed using ``key=value`` format into a dictionary that is
-        then returned.
+        then returned. If there is no `=` in the argument, value is an empty
+        string.
+
+        .. versionchanged:: 1.13.0 Support to `as_dictionary=True` and args
+        without `=`, generating an empty string instead of crashing.
```

Comment:
**Michael Bayer** (zzzeek) wrote:

needs to be indented:

        .. versionchanged:: 1.13.0 Support to `as_dictionary=True` and args
           without `=`, generating an empty string instead of crashing.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4989